get_logs.py

```
@@ -0,0 +1,14 @@
+import urllib.request
+import json
+import zipfile
+import io
+import os
+
+url = "https://api.github.com/repos/strawgate/memagent/actions/runs/24439395673/jobs"
+req = urllib.request.Request(url)
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)